build-packages.sh

```diff
@@ -11,8 +11,36 @@ working_tree_root="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 build_packages_log=$working_tree_root/build-packages.log
 binclashlog=$working_tree_root/binclash.log
 binclashloggerdll=$working_tree_root/Tools/Microsoft.DotNet.Build.Tasks.dll
+RuntimeOS=ubuntu
 
-options="/m /nologo /v:minimal /clp:Summary /flp:v=detailed;Append;LogFile=$build_packages_log /l:BinClashLogger,$binclashloggerdll;LogFile=$binclashlog"
+# Use uname to determine what the OS is.
+OSName=$(uname -s)
+case $OSName in
```

Comment:
@weshaggard I agree on your proposal. I opened the issue #7652 to keep track of it.